CONTRIBUTING.md

```diff
@@ -1,65 +1,120 @@
-    To test for both Python 2 and 3, use `tox` .
-
-2. Update the documentation
-    
-    Ensure any new features or behavioral differences introduced by your changes are documented in the docs, and ensure you have docstrings on your contributions. We use the Sphinx's Napoleon extension to build documentation from Google-style docstrings (see http://www.sphinx-doc.org/en/master/ext/napoleon.html).
-
-3. Sign the Contributor License Agreement
-
-    **When you contribute code, you affirm that the contribution is your original work and that you license the work to the project under the project’s open source license. Whether or not you state this explicitly, by submitting any copyrighted material via pull request, email, or other means you agree to license the material under the project’s open source license and warrant that you have the legal authority to do so.**
-
-    Please make sure you have signed our Contributor License Agreement (either [Individual Contributor License Agreement v1.0](https://docs.google.com/forms/d/e/1FAIpQLSdA-aWKQ15yBzp8wKcFPpuxIyGwohGU1Hx-6Pa4hfaEbbb3fg/viewform?usp=sf_link) or [Software Grant and Corporate Contributor License Agreement ("Agreement") v1.0](https://docs.google.com/forms/d/e/1FAIpQLSf3RZ_ZRWOdymT8OnTxRh5FeIadfANLWUrhaSHadg_E20zBAQ/viewform?usp=sf_link)). We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
-    
-4. Rebase your changes
-
-    Update your local repository with the most recent code from the main Great Expectations repository, and rebase your branch on top of the latest `develop` branch. We prefer small, incremental commits, because it makes the thought process behind changes easier to review.
-
-5. Submit a pull request
+## Testing
 
-    Push your local changes to your forked copy of the repository and submit a pull request. In the pull request, choose a title which sums up the changes that you have made, and in the body provide more details about what your changes do. Also mention the number of the issue where discussion has taken place, eg "Closes #123".
+### Running tests
+In general, you can run all tests by simply running `pytest` in the great_expectations directory root. Before submitting any PR, please ensure there are no tests that fail with an error, and review any warnings.
 
-6. Participate in review
+If you would like to run the tests without the need for local backend setups, adding the following flags will skip the respective backends:
+- `--no-postgresql` will skip postgres tests
+- `--no-spark` will skip spark tests 
+- `--no-sqlalchemy` will skip sqlalchemy tests
```

Comment:
Two small clarifications here: 
1. even with all of these flags set, the *pandas* backend *will* be tested...it's not currently possible to disable those tests.
2. `no-sqlalchemy` is a superset of `no-postgresql`...if you set no-sqlalchemy then no-postgresql is implied.